Scripts/RemediationScripts/Remediate-AnonymousAccessOnContainers.ps1

```diff
@@ -693,12 +823,12 @@ function Set-AnonymousAccessOnContainers
     )
 
     Write-Host $([Constants]::DoubleDashLine)
```

Comment:
Noramlly ... three dots represent something in progress, so I will suggest lets keep them as any how rollback will not be run from UI generated script. Thoughts?